mod/consensus-types/pkg/types/payload_test.go

```diff
@@ -89,7 +103,15 @@ func TestExecutableDataDeneb_GetTree(t *testing.T) {
 
 func TestExecutableDataDeneb_Getters(t *testing.T) {
 	payload := generateExecutableDataDeneb()
-
+	transactions := make([][]byte, 1)
+	transactions[0] = make([]byte, 0)
```

Comment:
Is this needed here ? I don't see it is used.

Reply:
I use this to make sure the logic when this array has elements works

Reply:
Here the logic that it will cover

Reply:
Here is the logic that this will cover err = ssz.UnmarshalDynamic(buf, num, func(indx int, buf []byte) (err error) {
			if len(buf) > 1073741824 {
				return ssz.ErrBytesLength
			}
			if cap(e.Transactions[indx]) == 0 {
				e.Transactions[indx] = make([]byte, 0, len(buf))
			}
			e.Transactions[indx] = append(e.Transactions[indx], buf...)
			return nil
		})

Reply:
Here is the line that it covers https://github.com/berachain/beacon-kit/blob/main/mod/consensus-types/pkg/types/payload_header.ssz.go#L217-L227